### PR TITLE
Added functionality for plotting step plots

### PR DESCRIPTION
**Pull request by edmundus**
Adapted the `draw_line2d` function to be able to correctly plot step plots created with `plt.step(where='')`.

The where argument can handle the styles `drawStyleKeys = ['default', 'steps-mid', 'steps-pre', 'steps-post', 'steps']` from [https://matplotlib.org/3.1.0/api/_as_gen/matplotlib.lines.Line2D.html](url)

Keywords for PGF are from [http://pgfplots.sourceforge.net/pgfplots.pdf](url) Subsection 4.5.3 Constant Plots.

### CODE REVIEW

File: tikzplotlib/line2d.py [hunk 1]
```diff
@@ -31,6 +31,17 @@ def draw_line2d(data, obj):
     data, line_xcolor, _ = mycol.mpl_color2xcolor(data, color)
     addplot_options.append(line_xcolor)
 
+    # get draw style
+    drawstyle = obj.get_drawstyle()
+    if drawstyle is not (None or "default"):
```
Comment (nschloe) [inline tikzplotlib/line2d.py]: Better:
```python
if drawstyle in [None, "default"]:
    pass
elif ...
```
Also, a test would be nice. Not strictly required though, it's just that something could be destroyed in the future then.

**Comment by edmundus** [comment]
Where can I see the what should be changed due to linting?

**Comment by Aikhjarto** [comment]
According to '.travis.yml' the following commands are used for lint:
```
pylint tikzplotlib/
pylint test/*py
```
You can also see calls to `black` in 'Makefile' for auto-formatting files.

**Comment by nschloe** [comment]
Just check the link on the failed tests ("Details") and you'll see what's wrong. `make black` will help you.

**Comment by edmundus** [comment]
Thanks @Aikhjarto and @nschloe for helping me learn how to contribute!

**Comment by edmundus** [comment]
Closing was the wrong action I think

**Comment by nschloe** [comment]
Very nice addition, thank you!

**Pull request merged**